setup_helpers.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (C) 2009-2015 Barry A. Warsaw
+#
+# This file is part of setup_helpers.py
+#
+# setup_helpers.py is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, version 3 of the License.
+#
+# setup_helpers.py is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with setup_helpers.py.  If not, see <http://www.gnu.org/licenses/>.
+
+"""setup.py helper functions."""
+
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
Yeah, it's just a wget from a standard template file.  Since we only need one function from there, I think I'll just vendor that in and get rid of the setup_helper.py file.